src/hotspot/share/oops/constantPool.cpp

```diff
@@ -167,10 +167,22 @@ objArrayOop ConstantPool::resolved_references_or_null() const {
   if (_cache == NULL) {
     return NULL;
   } else {
-    return (objArrayOop)_cache->resolved_references();
+    return _cache->resolved_references();
   }
 }
 
+oop ConstantPool::resolved_references_at(int index) const {
```

Comment:
This function should be called `resolved_reference_at` - singular - as it returns one reference. Thanks.

Reply:
I have it resolved_references because that's the name of the array and I want to keep that as a string to search on.